src/command.rs

```diff
@@ -63,9 +60,7 @@ pub enum PhoenixCLICommand {
         amount: u64,
     },
     /// Mint both base and quote tokens to a recipient for a given market. Default amounts are 100_000_000_000 for base and 100_000_000 for quote.
-    /// Devnet only
```

Comment:
add some sort of note on mint token commands about how these are for markets associated w/ token faucet